### DISCUSSION TITLE
Prettier rules not being honored

### DISCUSSION THREAD

**Discussion by jaybarra** [radian-software/apheleia#189] [General]
I have a typescript project where I have a ".prettierrc" file in my root, but I'm not seeing prettier honor the settings when apheleia invokes it in a buffer. I'm seeing tabs being inserted when I'm explicitly setting that to false. However if I run prettier via my NPM scripts, the settings are being picked up.

**Comment by raxod502** [comment]
If you check `apheleia-formatters` you can see the exact commands that are getting run, how does that compare to what you have in your NPM scripts?
